src/eval_pt_center.py

```diff
@@ -0,0 +1,254 @@
+import argparse
+import sys
+import os
+import re
+import array
+os.environ["CUDA_VISIBLE_DEVICES"] = ""  # pylint: disable=wrong-import-position
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+from concurrent.futures import ProcessPoolExecutor
+import itertools
+from flarefly.data_handler import DataHandler
+from flarefly.fitter import F2MassFitter
+import yaml
+import ROOT
+from ROOT import TFile
+import awkward as ak
+sys.path.append(f"{os.path.dirname(os.path.abspath(__file__))}/../utils")
+from utils import logger
+from matplotlib import gridspec
+import uproot
+from multiprocessing import Pool, cpu_count
+ROOT.gROOT.SetBatch(True)
+
+def convert_fit_funcs(func_list):
+    converted_funcs = []
+    for func in func_list:
+        if func == "kGaus":
+            converted_funcs.append("gaussian")
+        elif func == "kLin":
+            converted_funcs.append("chebpol1")
```

Comment:
Hi @Marcellocosti ! Thank you very much for this very important update. Could you please add also the exponential function to this list? Since it is one of the pdfs we use most in the inv. mass fit procedure it can  be helpful